packages/appstore/src/components/wallet-radio-button/wallet-radio-button.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import classNames from 'classnames';
+
+type TProps = {
+    icon: string;
+    is_disabled: boolean;
+};
+
+const WalletRadioButton: React.FC<TProps> = ({ icon, is_disabled }) => {
```

Comment:
```
const WalletRadioButton = ({ icon, is_disabled }: TProps) => {
```
Please avoid using `React.FC`. It's gonna be deprecated soon.
 - https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/
 - https://backstage.io/docs/architecture-decisions/adrs-adr006